tests/unit/type/string.tcl

```diff
@@ -582,6 +582,19 @@ if {[string match {*jemalloc*} [s mem_allocator]]} {
       set err1
     } {*WRONGTYPE*}
 
+    test "SET with IFEQ conditional" {
```

Comment:
Could you also add test case for interaction with other parameters?

Reply:
added tests with xx, nx and get